test/parallel/test-buffer-alloc.js

```diff
@@ -464,6 +464,9 @@ assert.strictEqual(
 // Regression test for https://github.com/nodejs/node/issues/3496.
 assert.strictEqual(Buffer.from('=bad'.repeat(1e4), 'base64').length, 0);
 
+// Regression test for https://github.com/nodejs/node/issues/11987.
+assert.strictEqual(Buffer.from("w0  ", 'base64').length, 1);
```

Comment:
Thanks @seishun and @addaleax.

The test is good. Agree on checking the actual bytes:

```
assert.deepStrictEqual(Buffer.from('w0  ', 'base64'), Buffer.from([195]));
```